app/models/user.rb

```diff
@@ -19,7 +19,9 @@ def identifier
 
   # Unlock all active locks.
   def unlock_all!
+    unlocked_locks = locks
     locks.map(&:unlock!)
+    return unlocked_locks
```

Comment:
You can just do
``` ruby
def unlock_all!
  locks.each(&:unlock!)
end
```

each will return the array, map will return a new array of return values of the block executed

Reply:
Oh cool! yup, that was throwing me off, the return values (of the map block) was a list of `true` objects!